api/security/policy.go

```diff
@@ -0,0 +1,62 @@
+package security
+
+import (
+	"net/http"
+)
+
+// Various security headers
+const (
+	HeaderGatewaySource = "X-Ovh-Gateway-Source"
```

Comment:
What is this header for?

Reply:
Logging purpose, has now been moved to configuration